friends/src/App.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './App.css';
+import FriendList from '../../../../Lambda-School/workspace/Single-Page-Applications/day3/HTTP-AJAX/friends/src/components/FriendList';
```

Comment:
this import is messed up, should be `import FriendList from './components/FriendList';`